### PR TITLE
fix: Plain string was thrown when waiting for blockchain properties t…

### PR DESCRIPTION
**Pull request by JonaszJestem**
…o be deployed

### CODE REVIEW

File: packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts [hunk 1]
```diff
@@ -118,4 +110,16 @@ export class BlockchainPropertiesService {
...
+
+    private async waitForPropertiesToBeDeployed() {
+        try {
+            await waitForState(
+                async () => await this.isDeployed(),
+                'Blockchain properties were not deployed',
+                { interval: 5_000, maxTries: 24 }
+            );
+        } catch (e) {
+            throw Error(e);
```
Comment (soanvig) [inline packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts:122]: What the hell? Why it's not `new Error` and why it just rethrows? Is `waitForState` throwing just string (e.g. `throw "some error message"`) not wrapped with error object?

**Review by soanvig — CHANGES_REQUESTED** [review]

File: packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts (hunk 1 above)
Comment (JonaszJestem) [reply packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts:122]: > Why it's not new Error 
my bad
>  Is waitForState throwing just string 
yes

Comment (soanvig) [reply packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts:122]: So cannot waitForState throw normal error?

Comment (JonaszJestem) [reply packages/origin-247-certificate/src/onchain-certificate/blockchain-properties.service.ts]: Done

**Review by soanvig — APPROVED** [review]

**Pull request merged**